test/unit/instance/save.test.js

```diff
@@ -11,11 +11,11 @@ const sinon     = require('sinon');
 
 describe(Support.getTestDialectTeaser('Instance'), () => {
   describe('save', () => {
-    it('should disallow saves if no primary key values is present', async () => {
-      const Model = current.define('User', {});
-      const instance = Model.build({}, { isNewRecord: false });
+    it('is not allowed if the instance does not have a primary key defined', async () => {
+      const User = current.define('User', {});
+      const instance = User.build({});
```

Comment:
This one should preserve its "isNewRecord: false", because `.save` does two different things based on that property:
- it updates the existing entity in the DB if isNewRecord is false (PK mandatory)
- it inserts a new one in the DB if isNewRecord is true (PK optional)

```suggestion
      const instance = User.build({}, { isNewRecord: false });
```